### PR TITLE
[docs] Update lazy-loading.md

### PR DESCRIPTION
**Pull request by Alex-Sokolov**
Added comments and inverted conditions to reduce code nesting and simplify understanding

### CODE REVIEW

File: vuepress/guide/lazy-loading.md [hunk 1]
```diff
@@ -45,18 +45,25 @@ function setI18nLanguage (lang) {
...
-      return import(/* webpackChunkName: "lang-[request]" */ `@/lang/${lang}`).then(msgs => {
-        i18n.setLocaleMessage(lang, msgs.default)
-        loadedLanguages.push(lang)
-        return setI18nLanguage(lang)
-      })
-    } 
+export function loadLanguageAsync(lang) {
+  // If the same language
+  if (i18n.locale === lang) {
+    return Promise.resolve(lang)
```
Comment (exoego) [inline vuepress/guide/lazy-loading.md]: Don't we need to invoke `setI18nLanguage(lang)` this case?

**Comment by exoego** [comment]
Thanks for opening a PR !!
Please submit a PR to `dev` branch as documented in https://github.com/kazupon/vue-i18n/blob/dev/CONTRIBUTING.md#pull-request-guidelines
I updated a target branch of this PR.

You may also need to rebase onto `dev` to fix test failures on CircleCI.

File: vuepress/guide/lazy-loading.md (hunk 1 above)
Comment (Alex-Sokolov) [reply vuepress/guide/lazy-loading.md]: @kazupon can you take a look is in needed here?

Comment (kazupon) [reply vuepress/guide/lazy-loading.md]: I think we need `setI18nLanguage(lang)`.

**Comment by Alex-Sokolov** [comment]
Added `setI18nLanguage(lang)`

**Pull request merged**